M2/Macaulay2/d/CMakeLists.txt

```diff
@@ -146,7 +146,8 @@ add_library(M2-interpreter OBJECT ${CLIST} ${CXXLIST} ${TAGS}
   $<$<BOOL:${WITH_XML}>:xml-c.c xml-c.h>
   $<$<BOOL:${WITH_PYTHON}>:python-c.c>)
 
-target_link_libraries(M2-interpreter PRIVATE M2-supervisor Boost::regex
+target_link_libraries(M2-interpreter PRIVATE M2-supervisor
+  Boost::boost # boost_regex is now header-only, so we don't link Boost::regex
```

Comment:
Will this matter when building on older systems where it's not header-only?

Reply:
regex has been header-only since Boost 1.77 (~2021), so I'm not worried about it.